openedx/event-bus-redis
# feat: replicate event bus kafka

**Description:** Replicates [event_bus_kafka](https://github.com/openedx/event-bus-kafka) and removes kafka specific code. Cleans up producer and consumer to make it easier to add redis streams implementation.

_Since there are a lot of files added, it would be easier to review commit by commit._

**Tickets:** 
* `Private-ref`: [BB-7169](https://tasks.opencraft.com/browse/BB-7169)
* #1 


**Merge deadline:** List merge deadline (if any)

**Testing instructions:**

1. Make sure tests are passing.
2. Make sure kafka specific code is removed.

**Reviewers:**
- [ ] @keithgg
- [ ] @bmtcril 

**Merge checklist:**

**Post merge:**
      finished.
